src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card"
+      data-qa="card"
+    >
+
+      <div class="card__image">
+        <img
+        class="card__image-iMac"
+        src="images/imac.jpeg"
+        alt="iMac"
+        >
+      </div>
+
+      <div class="card__title">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__rate">
+        <div class="card__star stars-4">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
+        <p class="card__review">Reviews: 5</p>
+      </div>
+
+      <div>
+        <ul class="card__price-list">
+          <li class="card__price-title">Price:</li>
+          <li class="card__price-amount">$2,199</li>
+        </ul>
+      </div>
+
+      <div class="card__buy-button">
+        <a href="#"
+        class="card__buy-button-link"
```

Comment:
Please use proper BEM convention use '__' (double underscore) for separation of Block and element.
for example: 
Block --> product
1st element --> product__buy-button
2nd element --> product__button-link or product__buy-link.